<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="ZAP Test Suite" duration-ms="13120" started-at="2023-01-31T17:03:34Z" finished-at="2023-01-31T17:03:47Z">
    <groups>
    </groups>
    <test name="ZAP TEST" duration-ms="13120" started-at="2023-01-31T17:03:34Z" finished-at="2023-01-31T17:03:47Z">
      <class name="com.project.home.HomePageTest">
        <test-method status="PASS" signature="startAppiumServer(java.lang.String, java.lang.String)[pri:0, instance:com.project.home.HomePageTest@443dbe42]" name="startAppiumServer" is-config="true" duration-ms="100" started-at="2023-01-31T17:03:34Z" finished-at="2023-01-31T17:03:34Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[web]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ZAP_PROXY]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startAppiumServer -->
        <test-method status="PASS" signature="setupDriver(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.project.home.HomePageTest@443dbe42]" name="setupDriver" is-config="true" duration-ms="4635" started-at="2023-01-31T17:03:34Z" finished-at="2023-01-31T17:03:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[web]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ZAP_PROXY]]>
              </value>
            </param>
            <param index="2">
              <value is-null="true"/>
            </param>
            <param index="3">
              <value is-null="true"/>
            </param>
            <param index="4">
              <value is-null="true"/>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupDriver -->
        <test-method status="FAIL" signature="test()[pri:0, instance:com.project.home.HomePageTest@443dbe42]" name="test" duration-ms="5210" started-at="2023-01-31T17:03:39Z" finished-at="2023-01-31T17:03:44Z">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[unknown error: net::ERR_PROXY_CONNECTION_FAILED
  (Session info: chrome=109.0.5414.120)
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'ALIPL1537', ip: '192.168.0.107', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_202'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 109.0.5414.120, chrome: {chromedriverVersion: 109.0.5414.74 (e7c5703604da..., userDataDir: C:\Users\AL2436\AppData\Loc...}, goog:chromeOptions: {debuggerAddress: localhost:53826}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(manual, http=localhos..., setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: e58e40f759afb642eb43c0d916023e3e]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: unknown error: net::ERR_PROXY_CONNECTION_FAILED
  (Session info: chrome=109.0.5414.120)
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'ALIPL1537', ip: '192.168.0.107', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_202'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 109.0.5414.120, chrome: {chromedriverVersion: 109.0.5414.74 (e7c5703604da..., userDataDir: C:\Users\AL2436\AppData\Loc...}, goog:chromeOptions: {debuggerAddress: localhost:53826}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(manual, http=localhos..., setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: e58e40f759afb642eb43c0d916023e3e
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:277)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteNavigation.to(RemoteWebDriver.java:857)
at com.testautomation.ui.base.ElementEvent.gotoURL(ElementEvent.java:74)
at com.project.pages.home.HomePage.gotoSite(HomePage.java:19)
at com.project.home.HomePageTest.test(HomePageTest.java:18)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="PASS" signature="teardownDriver()[pri:0, instance:com.project.home.HomePageTest@443dbe42]" name="teardownDriver" is-config="true" duration-ms="639" started-at="2023-01-31T17:03:45Z" finished-at="2023-01-31T17:03:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownDriver -->
        <test-method status="FAIL" signature="stopAppiumServer(java.lang.String, java.lang.String)[pri:0, instance:com.project.home.HomePageTest@443dbe42]" name="stopAppiumServer" is-config="true" duration-ms="2037" started-at="2023-01-31T17:03:45Z" finished-at="2023-01-31T17:03:47Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[web]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ZAP_PROXY]]>
              </value>
            </param>
          </params>
          <exception class="org.zaproxy.clientapi.core.ClientApiException">
            <message>
              <![CDATA[java.net.ConnectException: Connection refused: connect]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.zaproxy.clientapi.core.ClientApiException: java.net.ConnectException: Connection refused: connect
at org.zaproxy.clientapi.core.ClientApi.callApiOther(ClientApi.java:447)
at org.zaproxy.clientapi.gen.Core.htmlreport(Core.java:860)
at com.testautomation.ui.security.ZapScan.generateZapReport(ZapScan.java:148)
at com.testautomation.ui.base.BaseTest.stopAppiumServer(BaseTest.java:194)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.afterRun(TestRunner.java:725)
at org.testng.TestRunner.run(TestRunner.java:509)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused: connect
at java.net.DualStackPlainSocketImpl.connect0(Native Method)
at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)
at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
at java.net.Socket.connect(Socket.java:589)
at java.net.Socket.connect(Socket.java:538)
at sun.net.NetworkClient.doConnect(NetworkClient.java:180)
at sun.net.www.http.HttpClient.openServer(HttpClient.java:463)
at sun.net.www.http.HttpClient$1.run(HttpClient.java:515)
at sun.net.www.http.HttpClient$1.run(HttpClient.java:513)
at java.security.AccessController.doPrivileged(Native Method)
at sun.net.www.http.HttpClient.privilegedOpenServer(HttpClient.java:512)
at sun.net.www.http.HttpClient.openServer(HttpClient.java:553)
at sun.net.www.http.HttpClient.<init>(HttpClient.java:242)
at sun.net.www.http.HttpClient.New(HttpClient.java:339)
at sun.net.www.http.HttpClient.New(HttpClient.java:357)
at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1220)
at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1199)
at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1050)
at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:984)
at org.zaproxy.clientapi.core.ClientApi.getConnectionInputStream(ClientApi.java:417)
at org.zaproxy.clientapi.core.ClientApi.callApiOther(ClientApi.java:432)
... 22 more
]]>
            </full-stacktrace>
          </exception> <!-- org.zaproxy.clientapi.core.ClientApiException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- stopAppiumServer -->
      </class> <!-- com.project.home.HomePageTest -->
    </test> <!-- ZAP TEST -->
  </suite> <!-- ZAP Test Suite -->
</testng-results>
